src/main/java/tech/gruppone/stalker/server/configuration/SmtpMailSenderConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package tech.gruppone.stalker.server.configuration;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SmtpMailSenderConfiguration {
+
+  @Autowired private JavaMailSender javaMailSender;
+
+  public void send(String to, String subject, String body) throws MessagingException {
+    MimeMessage message = javaMailSender.createMimeMessage();
+
+    // SSL Certificate.
+    MimeMessageHelper helper = new MimeMessageHelper(message, true);
+
+    // Multipart messages.
+    helper.setSubject(subject);
+    helper.setTo(to);
+    helper.setText(body, true);
+
+    javaMailSender.send(message);
+  }
```

Comment:
questo sarebbe più appropriato in un servizio (EmailService?) però dato che mi pare che lo usiamo solo in un punto (passwordrecovery) lascialo pure come metodo privato nel servizio passwordrecovery

Reply:
Sì lo usiamo solo in un punto, lo inserisco all'interno del service che vado a creare ora